tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -339,38 +454,53 @@ class MklFusedBatchNormBwdPrimitive : public MklPrimitive {
 
     context_.diff_src_mem->set_data_handle(static_cast<void*>(diff_src_data));
 
-    // execution
+#ifdef ENABLE_MKLDNN_V1
+    // Execute backward batch-normalization primitives.
+    DCHECK_EQ(context_.bwd_primitives.size(), context_.net_args.size());
+    for (size_t i = 0; i < context_.bwd_primitives.size(); ++i) {
+      context_.bwd_primitives.at(i).execute(*context_.bwd_stream,
+                                            context_.net_args.at(i));
+    }
```

Comment:
Please call `execute_primitives` in `mkl_util.h`.

Reply:
This hasn't been addressed yet.

Reply:
Will address this later